src/components/auth.js

```diff
@@ -18,6 +19,23 @@ const Auth = () => {
         };
     };
 
+    const processMicrosoftLogin = (response, data) => {
+        if(response.status !== 'unknown') {
+            dispatch({ 
+                type: LOGIN, 
+                name: data.name,
```

Comment:
Is there more data here... similar to what you have from Facebook? Is there a picture?

Reply:
Picture not available through Microsoft